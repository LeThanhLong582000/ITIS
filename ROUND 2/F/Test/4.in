100
951 343 697
248 460 750
909 441 713
783 416 653
997 284 826
25 582 831
195 578 372
167 892 573
918 668 957
97 473 259
38 509 304
633 779 910
864 525 88
19 240 448
633 1000 376
392 672 547
309 412 306
34 323 108
93 836 733
534 132 137
398 407 499
212 588 919
764 43 605
584 593 153
948 5 752
17 800 350
841 718 122
895 247 185
962 185 780
202 870 101
73 95 984
549 109 439
528 982 640
402 418 519
956 935 139
996 2 766
576 559 10
905 979 240
829 22 355
334 38 867
4 485 307
308 325 839
704 786 557
486 217 662
434 320 582
230 416 870
264 789 590
849 602 935
694 700 922
633 799 71
319 250 44
379 21 326
569 781 490
175 870 150
928 196 114
633 367 639
468 533 397
292 789 483
513 32 596
305 995 859
19 637 118
234 617 781
669 255 69
404 621 40
670 286 460
582 665 765
422 847 252
678 119 705
218 972 132
624 457 385
179 97 133
249 207 510
961 445 389
728 556 324
610 93 384
657 148 702
457 455 525
875 357 249
581 609 719
124 662 154
309 597 425
529 439 211
420 530 425
111 607 728
695 892 612
314 839 594
333 24 150
893 285 293
422 93 404
689 890 769
299 555 353
118 817 614
322 130 655
382 671 2
362 493 510
448 309 586
591 981 607
93 846 33
701 292 947
568 643 651
