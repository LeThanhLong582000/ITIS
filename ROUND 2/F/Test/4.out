653
269
591
510
770
137
0
0
604
0
0
382
89
114
5
134
102
0
0
138
245
272
606
72
753
0
123
186
779
0
481
440
93
252
80
767
11
83
356
582
0
411
238
466
348
342
33
591
458
0
45
327
139
0
115
453
166
0
539
85
0
199
70
0
422
341
0
632
0
276
108
276
390
248
385
606
264
250
346
0
69
151
158
116
208
35
151
294
367
284
49
0
424
0
190
363
109
0
678
288
